rails/Dockerfile

```diff
@@ -56,6 +66,9 @@ RUN \
   && cp ./config/app_environment_variables.sample.rb app_environment_variables.rb \
   && cp -r ./docker/prod/config/initializers/ ./config/
 
+# Change to the react-components directory and run npm build BEFORE rails assets precompile
+RUN cd react-components && npm install && npm run build && cd ..
```

Comment:
Shouldn't this use `npm ci` instead of `npm install` so that it uses the lockfile?